sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -50,6 +52,39 @@
 from apache_beam.utils.timestamp import Timestamp
 
 
+class CounterIncrememtingCombineFn(beam.CombineFn):
```

Comment:
Maybe call this SortedConcatWithCounters?

Reply:
Done.